atef/widgets/config.py

```diff
@@ -902,8 +902,10 @@ def handle_locking(self, locked: bool):
             self.delete_button.setEnabled(False)
         else:
             self.desc_edit.setFrameShape(self.desc_edit.StyledPanel)
+            color = self.palette().color(QPalette.ColorRole.Base)
             self.setStyleSheet(
-                "QLineEdit, QPlainTextEdit { background: white }"
+                f"QLineEdit, QPlainTextEdit {{ background: rgba({color.red()},"
```

Comment:
It took me a while to find all the options for color specification since I was curious where the rgba thing came from: https://doc.qt.io/qt-5/stylesheet-reference.html#list-of-property-types

```
rgb(r, g, b)
| rgba(r, g, b, a)
| hsv(h, s, v)
| hsva(h, s, v, a)
| hsl(h, s, l)
| hsla(h, s, l, a)
| #rrggbb
| [Color Name](https://doc.qt.io/qt-5/qcolor.html#setNamedColor)
```

Should we include the `color.alpha()` channel? It seems like a arbitrary omission

Reply:
In the first proposal it was just set to `255`, then removed, and now set to the palette's alpha value as in 744cc49. I think the latter is the most correct, so let's go with it.

Reply:
Didn't see this comment until now- my bad!